js/ui/list/ui.list.base.js

```diff
@@ -700,6 +700,11 @@ var ListBase = CollectionWidget.inherit({
         this.callBase(newItems);
     },
 
+    _refreshContent: function() {
+        this._prepareContent();
```

Comment:
`this.callbase()` ? The logic from parent class is not applied.

Reply:
_refreshContent of the collectionWidget call _renderContent. But this method (_renderContent) of the list call only _fireContentReadyAction. In this PR I try move some logic of the content rendering in the _initMarkup. I do it only for list, but not for other collections.